comparisons/serializers.py

```
@@ -0,0 +1,84 @@
+        model = ComparisonCompany
+        fields = ['company']
+
+    # [핵심] 한 개씩 추가할 때도 5개 제한 체크
+    def validate(self, data):
+        comparison = self.context.get('comparison')
+        company = data.get('company')
+
+        if comparison:
+            #  1. 중복 체크
+            if ComparisonCompany.objects.filter(comparison=comparison, company=company).exists():
+                raise serializers.ValidationError("이미 이 매치업에 등록된 기업입니다.")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)